src/components/Button/Button.jsx

```diff
@@ -1,76 +1,86 @@
 // @flow
 import cx from 'classnames';
 import React, { type Node } from 'react';
-import { getDataAttrs } from '../../dataUtils';
+
 import Loader from '../Loader/Loader.jsx';
-import style from './style.scss';
+
+import { getDataAttrs } from '../../dataUtils';
+
 import type { Data } from '../../types';
 
-const variants = { PRIMARY: 'primary', SECONDARY: 'secondary', TERTIARY: 'tertiary' };
-const size = { SMALL: 'small' };
+import style from './style.scss';
+
+const Variants = { PRIMARY: 'primary', SECONDARY: 'secondary', TERTIARY: 'tertiary' };
+const Sizes = { SMALL: 'small' };
```

Comment:
In the future, it might be a good idea to remove the "small" option. It is not in the Plasma design docs.

Reply:
I agree.